ibm/data_source_ibm_is_flow_log.go

```diff
@@ -206,25 +206,18 @@ func dataSourceIBMIsFlowLogRead(context context.Context, d *schema.ResourceData,
 	var flowLogCollector *vpcv1.FlowLogCollector
 
 	if name != "" {
-		start := ""
 		allrecs := []vpcv1.FlowLogCollector{}
-		for {
-			listOptions := &vpcv1.ListFlowLogCollectorsOptions{
-				Name: &name,
-			}
-			if start != "" {
-				listOptions.Start = &start
-			}
-			flowlogCollectors, response, err := sess.ListFlowLogCollectors(listOptions)
-			if err != nil {
-				return diag.FromErr(fmt.Errorf("Error Fetching Flow Logs for VPC %s\n%s", err, response))
-			}
-			start = GetNext(flowlogCollectors.Next)
-			allrecs = append(allrecs, flowlogCollectors.FlowLogCollectors...)
-			if start == "" {
-				break
-			}
+
+		listOptions := &vpcv1.ListFlowLogCollectorsOptions{
+			Name: &name,
 		}
+
+		flowlogCollectors, response, err := sess.ListFlowLogCollectors(listOptions)
+		if err != nil {
+			return diag.FromErr(fmt.Errorf("Error Fetching Flow Logs for VPC %s\n%s", err, response))
+		}
+		allrecs = append(allrecs, flowlogCollectors.FlowLogCollectors...)
```

Comment:
i think we can remove allrecs as well. and add the condition if len(flowlogCollectors) == 0
this applies for all the cases below.

Reply:
`flowlogCollectors` is of type *vpcv1.FlowLogCollectorCollection

Reply:
something like this 
if len(flowlogCollectors.FlowLogCollectors) != 0 {
	flowLogCollector = &flowlogCollectors.FlowLogCollectors[0]
}